src/config.rs

```diff
@@ -18,6 +18,10 @@ pub struct ConfigFile {
     #[serde(flatten)]
     pub identity: Option<Identity>,
 
+    /// Path to database file.
+    #[serde(default = "default_db_path")]
+    pub db_path: PathBuf,
```

Comment:
Instead of this, we should default to *not have* persistent storage at all.

The easiest way is to make this an Option, and have the DATABASE an Option too, and have db_read and db_write be noops if there's no actual database